cypress/integration/organizations/user-integration-organization.spec.js

```diff
@@ -0,0 +1,85 @@
+import TopMenu from '../../support/fragments/topMenu';
+import permissions from '../../support/dictionary/permissions';
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import users from '../../support/fragments/users/users';
+import newOrganization from '../../support/fragments/organizations/newOrganization';
```

Comment:
Please import fragment starting from the UpperCase letter

Reply:
Done